source/get-started.txt

```diff
@@ -49,8 +49,13 @@ Download and Install
       Before you begin this tutorial, ensure you have the following dependencies
       installed in your development environment:
       
-      - Compiler that supports C++17, such as `GCC <https://gcc.gnu.org/install/>`__, `Clang <https://clang.llvm.org/>`__,
-        or `Visual Studio <https://visualstudio.microsoft.com/>`__
+      - Compiler that supports C++17, such as one of the following:
+
+         - `GCC <https://gcc.gnu.org/install/>`__ v8.1 or newer
+         - `Clang <https://clang.llvm.org/>`__ v3.8 or newer
+         - `Apple Clang <https://developer.apple.com/xcode/>`__ v13.1 or newer
+         - `Visual Studio <https://visualstudio.microsoft.com/vs/features/cplusplus/>`__ 2015 Update 3 or newer
```

Comment:
S: I'd [use "later"](https://www.mongodb.com/docs/meta/style-guide/terminology/general-term-guidelines/use-consistent-time-references/) instead of "newer":
```suggestion
         - `GCC <https://gcc.gnu.org/install/>`__ v8.1 or later
         - `Clang <https://clang.llvm.org/>`__ v3.8 or later
         - `Apple Clang <https://developer.apple.com/xcode/>`__ v13.1 or later
         - `Visual Studio <https://visualstudio.microsoft.com/vs/features/cplusplus/>`__ 2015 Update 3 or later
```